guides/ssl-certificates/route53.md

```diff
@@ -104,7 +104,7 @@ To make sure that your `clusterIssuer` can change your DNS settings,
        preferredChain: ""
        privateKeySecretRef:
          name: example-issuer-account-key
-       server: https://acme-staging-v02.api.letsencrypt.org/directory
+       server: https://acme-v02.api.letsencrypt.org/directory
```

Comment:
Should we provide examples for both? I imagine that someone would want to test with staging first, and then switch to production once they confirm it works, because I think the rate limits are different between staging and prod?

Reply:
Correct - the rate limits are different. [See here](source /tmp/pure_installer.fish; and install_pure). I'm in favor of including an example for each, so users are aware of both options and can decide for themselves.